packages/cli/src/__tests__/ceramic-daemon.test.ts

```diff
@@ -389,6 +389,30 @@ describe('Ceramic interop: core <> http-client', () => {
       expect(resCore[docC.id.toString()].content).toEqual(resClient[docC.id.toString()].content)
       expect(resCore[docD.id.toString()].content).toEqual(resClient[docD.id.toString()].content)
     })
+
+    it('returns stream when using multiquery with genesis and no updates', async () => {
+      const metadata = {
+        controllers: ['did:test'],
+        family: 'test',
+      }
+      const genesis = await TileDocument.makeGenesis(
+        {
+          did: core.did,
+        },
+        null,
+        {
+          ...metadata,
+          deterministic: true,
```

Comment:
nitpick: why not just put this in the original `metadata` object?

Reply:
Because `deterministic` is not part of the metadata when we receive the query response

```
expect(resCore[streamId.toString()].metadata).toEqual(metadata)
expect(resClient[streamId.toString()].metadata).toEqual(metadata)
```

Reply:
ah yes, good call.  Sounds good